projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-jitm-v2.php

```diff
@@ -0,0 +1,151 @@
+<?php
+/**
+ * REST API endpoint for retrieving JITMs from the WPCOM API.
```

Comment:
@scruffian Do you agree we should pull this out into a separate PR?

That way the _code_ can be available on Simple, Atomic and JP and we can enable it on Simple by way of 170707-ghe-Automattic/wpcom when we are ready.

Also we can write some tests for it.

Reply:
Yeah good idea

Reply:
https://github.com/Automattic/jetpack/pull/41993